src/multi_vector_simulator/C0_data_processing.py

```diff
@@ -1776,22 +1777,30 @@ def process_maximum_cap_constraint(dict_values, group, asset, subasset=None):
     else:
         asset_dict = dict_values[group][asset][subasset]
 
-    # Check if a maximumCap is defined
+    # include the maximumAddCap parameter to the asset dictionary
+    asset_dict.update({MAXIMUM_ADD_CAP: {VALUE: None}})
```

Comment:
Saw that you add the unit at the end of the function. You could also do it like this:

`asset_dict.update({MAXIMUM_ADD_CAP: {VALUE: None, UNIT: None}})`

And later in the if-statements only update the value:

`asset_dict[MAXIMUM_ADD_CAP].update({VALUE: None})`